.github/workflows/nodejs.yml

```
@@ -24,6 +24,6 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - run: yarn install --frozen-lockfile
-      - run: yarn tsc -p . --noEmit
+      - run: yarn test
```

Why you removed the compilation task? I think you should add this after the compilation task.
```suggestion
      - run: yarn tsc -p . --noEmit
      - run: yarn test
```